Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -1647,6 +1649,13 @@ public IEnumerable<BaseData> GetNextTicks()
 
             foreach (var tick in ticks)
             {
+                // if we have a cached data time zone for this symbol, perform the conversion 
+                DateTimeZone dataTimeZone;
+                if (_dataTimeZones.TryGetValue(tick.Symbol.Value, out dataTimeZone))
+                {
+                    tick.Time = tick.Time.ConvertTo(TimeZones.NewYork, dataTimeZone);
```

Comment:
Perhaps instead of double converting we should make "GetBrokerageTime()" return UTC instead of NY time?  Then this method would just convert from UTC -> Data Timezone we specify which makes more sense than hard coding NY-time anywhere.

It seems like we've gotten away with the "if-forex-then-est" statement because we only had 2 assets. We could make it all UTC then remove this part?